server/configuration.go

```diff
@@ -118,15 +124,37 @@ func (p *Plugin) setConfiguration(configuration *configuration) {
 
 // OnConfigurationChange is invoked when configuration changes may have been made.
 func (p *Plugin) OnConfigurationChange() error {
-	var configuration = new(configuration)
+	var cfg = new(configuration)
 
 	// Load the public configuration fields from the Mattermost server configuration.
-	if err := p.API.LoadPluginConfiguration(configuration); err != nil {
+	if err := p.API.LoadPluginConfiguration(cfg); err != nil {
 		return errors.Wrap(err, "failed to load plugin configuration")
 	}
 
-	p.setConfiguration(configuration)
-	p.zoomClient = zoom.NewClient(configuration.ZoomAPIURL, configuration.APIKey, configuration.APISecret)
+	if err := p.registerSiteURL(); err != nil {
+		return errors.Wrap(err, "could not register site URL")
+	}
+
+	p.setConfiguration(cfg)
+	p.jwtClient = zoom.NewJWTClient(cfg.ZoomAPIURL, cfg.APIKey, cfg.APISecret)
```

Comment:
Here I would use getZoomAPIURL or anything similar. Keep in mind that if ZoomAPIURL is empty, this will stop working.